### ISSUE TITLE
Button - Content considerations seem to suggest buttons for CTAs that perhaps should be links

### ISSUE THREAD

**Issue by humancompanion-usds** [department-of-veterans-affairs/vets-design-system-documentation#2013]
## Description
The Content considerations for button need to be reviewed:
https://design.va.gov/components/button/#content-considerations

This content is shared with the Content Style Guide page for Button Labels:
https://design.va.gov/content-style-guide/button-labels

The last section in particular has a bunch of examples that all seem, to me anyway, to be navigation and thus should use a LInk - Action, rather than a button:

> Always take users to the right level of access for the CTA.
> ​ Like this: “Explore VA health care” button should take you to a general health care benefits landing page like va.gov/health-care.
> ​ Like this: “Apply for VA health care” button should take you to an application page like va.gov/health-care/apply.
>​ Like this: “Compare GI Bill benefits” button should take you right to the comparison tool, va.gov/gi-bill-comparison-tool.

I'd like to either excise this or borrow from our Button vs. Link guidance and explain why these should not be buttons.

## Tasks
- [ ] Update component documentation for design.va.gov
- [ ] Request documentation draft review
- [ ] Once documentation is approved, publish to design.va.gov

## Acceptance Criteria
- [ ] Button documentation is updated on design.va.gov

**Comment by humancompanion-usds** [comment]
@laurenvocke-va - This is a suggested change in the content style guide so wanted to bring this to your attention.

**Comment by laurenvocke-va** [comment]
@humancompanion-usds Thank you for flagging this!

**Comment by laurenvocke-va** [comment]
@humancompanion-usds I reviewed this with @DanielleThierryUSDSVA and we agree that we should remove these examples from the button pages in the style guide and components. I'll get started on the PR today. Thanks!

**Comment by humancompanion-usds** [comment]
Fixed in https://github.com/department-of-veterans-affairs/vets-design-system-documentation/pull/4295
